projects/dynamicforms/src/core/services/builder/form-builder.service.ts

```diff
@@ -66,7 +66,7 @@ export class FormBuilderService {
     this.userAccountFacade
       .get()
       .subscribe(user => {
-        if ((<any>user).roles.includes('sellergroup')) {
+        if ((<any>user)?.roles.includes('sellergroup')) {
```

Comment:
Has nothing to do with this task, just a fix for some cases when user doesn't have `sellergroup`